### PR TITLE
Adds region AMI defaults

### PR DESCRIPTION
**Pull request by sthulb**
This specifies a default ami for all regions.

Refs: #287

### CODE REVIEW

File: drivers/amazonec2/utils.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	var list []string
+
+	for k := range regionDetails {
+		list = append(list, k)
+	}
+
+	return list
+}
+
+func validateAwsRegion(region string) (string, error) {
```
Comment (ehazlett) [inline drivers/amazonec2/utils.go]: Is this used anywhere?

File: drivers/amazonec2/utils.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	"eu-west-1":      {"ami-823686f5"},
+	"eu-central-1":   {"ami-ac1524b1"},
+	"sa-east-1":      {"ami-c770c1da"},
+	"us-east-1":      {"ami-4ae27e22"},
+	"us-west-1":      {"ami-d1180894"},
+	"us-west-2":      {"ami-898dd9b9"},
+	"us-gov-west-1":  {"ami-cf5630ec"},
+}
+
+func awsRegionsList() []string {
```
Comment (ehazlett) [inline drivers/amazonec2/utils.go]: Is this used anywhere?

File: drivers/amazonec2/utils.go (hunk 1 above)
Comment (sthulb) [inline drivers/amazonec2/utils.go]: No, it should be though.

**Comment by sthulb** [comment]
Fixed :) With tests.

File: drivers/amazonec2/utils.go (hunk 2 above)
Comment (ehazlett) [inline drivers/amazonec2/utils.go]: Is this needed?

Comment (sthulb) [inline drivers/amazonec2/utils.go]: It's used by the validate func below.

Comment (ehazlett) [inline drivers/amazonec2/utils.go]: Ah thx -- missed it :)

**Comment by ehazlett** [comment]
LGTM

**Pull request merged**